src/map/orientation.c

```diff
@@ -258,7 +258,7 @@ void map_orientation_update_buildings(void)
                     }
                 }
                 map_building_tiles_add(i, b->x, b->y, b->size, image_id, TERRAIN_GATEHOUSE | TERRAIN_BUILDING);
-                map_terrain_add_gatehouse_roads(b->x, b->y, b->subtype.orientation);
+                map_terrain_add_gatehouse_roads(b->x, b->y, 0);
```

Comment:
Do we need the function call at all?

Reply:
Err.... In my defense, I had about three days experience with Julius when I wrote that change.

Reply:
No problem, I'm just thinking out loud here :)

Reply:
Turns out the function is actually needed to add the roads *under* the gatehouse, otherwise nothing can cross it, not even destination walkers.

Reply:
Ah yes, I see. And `0` is not a valid orientation so the extra roads will not be added.